devops/create_https_wss_endpoints_aws/helpers.py

```diff
@@ -16,13 +16,59 @@
 if args.network_value:
     network_config = current_work_path + '/' + args.network_value + '.json'
 
+network_config_cache = {}
+
+
+def parse_network_abbr(file_path):
+    global network_config_cache
+    # load network config
+    with open(file_path, 'r') as f:
+        network_config_dict = json.load(f)
+    network_config_cache = network_config_dict
+
+    # analyze network type
+    domain_name = network_config_dict['domain_name']
+    domain_name_parts = domain_name.split('.')
+    network_abbr = domain_name_parts[0]
+    return network_abbr
+
+
+def parse_distribution_config(file_path):
```

Comment:
kindof overkill to create a dict and method for this task imo. We just need to `grep` the explorer IP from the `distribution_config.txt` file and plug them in the main method.

Reply:
This also further simplify the master template.